docs/wifi-menu.8.txt

```diff
@@ -0,0 +1,89 @@
+WIFI-MENU(8)
+============
+
+NAME
+----
+wifi-menu - Interactively connect to a wireless network
+
+
+SYNOPSIS
+--------
+*wifi-menu* [-o | --obscure] [+INTERFACE+]
+
+*wifi-menu* [-h | --help]
+
+
+DESCRIPTION
+-----------
+*wifi-menu* allows a user to interactively connect to a wireless network
+over INTERFACE using an existing netctl profile, or an automatically
+generated profile for basic configurations. If only one wireless
+interface is available, INTERFACE can be omitted.  If no interface is
+specified and more than one is detected, a choice of interfaces is
+offered.
+
+*wifi-menu* is only able to generate netctl profiles for simple network
+configurations. Profiles for more complex configurations must be set up
+manually, or through the use of other tools.
+
+OPTIONS
+-------
+
+*-h, --help*::
+    Print a short help text and exit.
+
+*-o, --obscure*::
+    Do not echo password, and store derived pre-shared key
+    in profile instead of plaintext password.
```

Comment:
> I'm fine with "do not echo password". The "Show asterisks..." bit is just cut-and-paste from the output of --help. ;)

Ah, thanks for this remark! That reminds me of why I chose this wording. The 'do not echo' wording applies more to what you see for instance when you type your password on a login terminal (i.e. nothing at all).
Let's revert to something that closely resembles your original idea.

```suggestion
    Show asterisks for the characters of the password and store the password
    as a hexadecimal pre-shared key.
```